packages/verification-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@certifaction/verification-core",
-    "version": "2.0.0",
+    "version": "2.0.1",
```

Comment:
You don't need to change this by yourself. After it's merged to master you can publish the package with lerna which then changes the version numbers.

Reply:
Ah okay, thanks.